int32 user_input
---
int32 return

